language/compiler/ir-to-bytecode/src/compiler.rs

```diff
@@ -565,6 +565,8 @@ pub fn compile_module<'a>(
         constant_pool,
         struct_defs,
         function_defs,
+        // TODO
```

Comment:
Maybe tag yourself in it? Not needed, but I find that helpful sometimes. I do `// TODO(tmn)`

Regardless, a short description is nice incase this is missed and someone comes and reads the code later

Reply:
It's actually already done, but yes, I can elaborate the TODO and tag myself.